content/creating-user-centred-content/content-design/methods.md

```diff
@@ -14,7 +14,7 @@ Good content design begins with user research. This is then translated into simp
 
 A good content designer always asks what the user need is and how to meet this need. Be careful not to exclude user groups or assume what services users might need. Always think about [inclusivity and diversity](/user-research/identifying-users-needs/#make-your-research-inclusive).
 
-To create user-centred content you need to [research your users](/creating-user-centred-content/using-existing-research), which includes [using existing research](/content-strategy/creating-user-centred-content/using-existing-research/).
+To create user-centred content you need to [research your users](/creating-user-centred-content/using-existing-research), which includes [using existing research](/creating-user-centred-content/using-existing-research/).
```

Comment:
These links are broken. Suggest also `research your users` should be `research with your users`. Liz OKed this change in the Google Doc.